code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -55,10 +55,13 @@
 				span_danger("The [bullet.name] gets reflected by [src]!"),
 				span_userdanger("The [bullet.name] gets reflected by [src]!"),
 			)
-			// Finds and plays the block_sound of item which reflected
 			for(var/obj/item/held_item in held_items)
+				// Plays the block_sound of the item which reflected
 				if(held_item.IsReflect(def_zone))
 					playsound(src, held_item.block_sound, BLOCK_SOUND_VOLUME, TRUE)
+				// Applies a stamina penalty if the reflect item asks for it
+				if(held_item.IsReflectCostly())
+					apply_damage(REFLECT_STAMINA_COST, STAMINA)
```

Comment:
hey so there are these cool things called hulks

Reply:
Hello! I know hulks exist, my preferred method of take-down is a mutadone syringe supplied by the CMO.
I don't see how this is relevant, sorry. Using security gear to try and take down a hulk is really questionable and a strange stance to use for arguing.

Reply:
You cannot expect security to fully rely on another department to take down a hulk, not to mention hulk's power disparity cannot be that high considering its a mutation that only humans can have.

To allow hulks to use something that synergizes with them is to buff hulks and by extension humans.

Reply:
Just remove hulk. That's all I have left to add to this.

Reply:
she's actually cooking here removing hulk is good